src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller.Session.cs

```diff
@@ -64,12 +64,19 @@ internal void UpdateModelTrackingSpan(SnapshotPoint initialPosition)
                 });
             }
 
-            public override void Stop()
+            public override void DismissEditorSession()
             {
                 AssertIsForeground();
                 this.PresenterSession.ItemSelected -= OnPresenterSessionItemSelected;
                 this.PresenterSession.ItemCommitted -= OnPresenterSessionItemCommitted;
-                base.Stop();
+                base.DismissEditorSession();
+            }
+
+            public void CommitEditorSession()
```

Comment:
Sounds good.  However, if only one should be called, can we add state that tracks that and asserts/throws if someone does something wrong?